ports/views.py

```diff
@@ -277,6 +330,19 @@ def search_ports_in_maintainer(request):
 
 
 # Accept submissions from mpstats and update the users table
+def populate_os_table(user_obj):
```

Comment:
I would like to solve this in a different way, but that's a topic to be discussed separately. At the moment it's really awesome to see some actual statistics appear on the site.